### ISSUE TITLE
Questions and clarifications with the product backlog (previously "test")

### ISSUE THREAD

**Issue by andrewbutterfield** [lukeegan/GroupProject#1]
please respond

**Comment by lukeegan** [comment]
hello, closing this issue

**Comment by lukeegan** [comment]
We cannot assign a new issue to andrewbutterfield as you asked to not be marked as a collaborator on this repo, so we have to re-open this issue.

We found that the product backlog contained many stories that were duplicates (or very close to duplicates), or too vague to estimate. We attempted at getting clarifications in our meeting, but there was not sufficient time. 
These stories require further details to accurately estimate the time to complete the story. It is in the product owners interest to have the stories be clear and constrained, otherwise assumptions will be made, and cannot be changed in the future by the product owner. 
With the first sprint already more than 25% over, we hope that a final sprint backlog can be provided soon.

| Number | Story                                                                                                                                                     | Notes                                                                                                                                                                                                                                                                                 | 
|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
| 3      | "I would like to comply with EU data protection laws, so I don‰ŰŞt get fined heftily"                                                                     | "A very high value, as this is a task that should probably be overseen by a lawyer (which will not be paid for). Aside from the obvious security issues, that we would deal with on an ongoing basis."                                                                                | 
| 12     | I need to be able to manage  the responses so I can know who is attending                                                                                 | "Vague, what does ""manage the responses"" mean, therefore we cannot currently estimate this time  accuratly"                                                                                                                                                                         | 
| 15     | "Set up a ticketing service, so people can buy tickets without human interaction"                                                                         |                                                                                                                                                                                                                                                                                       | 
| 16     | e-ticketing for users                                                                                                                                     | "What does this mean, is this the same as above (number 15)?"                                                                                                                                                                                                                         | 
| 18     | Online ticketing ‰ŰŇ keeping in mind that mostly repeat customers/attendees                                                                               | "Is this the same as number 16? If not, what aspect of this story is different to 16 or 15?"                                                                                                                                                                                          | 
| 19     | "Keep track of who arrives on the night -- QR, invoices, receipts (linked to mailing lists)"                                                              | "What does ""keep track of"" mean? Does this refer to pulling up the profile of an attendee on the night?"                                                                                                                                                                            | 
| 20     | I would like to email (legitimately) subscribed users                                                                                                     | "What does ""legitimately"" refer to in this case? How are we supposed to know?"                                                                                                                                                                                                      | 
| 26     | I want to be able to manage attendees present at the event                                                                                                | "What does ""manage attendees"" refer to? Is this the same as number 19?"                                                                                                                                                                                                             | 
| 27     | I want to be able to manage the guest list                                                                                                                |                                                                                                                                                                                                                                                                                       | 
| 28     | I want to be able to add/remove attendees from the guest list                                                                                             | "Is this the same story as above (number 27)? If not, in which ways are they different?"                                                                                                                                                                                              | 
| 29     | I want to create menus                                                                                                                                    | Are these menus to show to attendees?                                                                                                                                                                                                                                                 | 
| 31     | I want to be able to share special dietary requirements (including table info for guest) with the caterers so that they can make appropriate arrangements | "How do caterers want to be contacted? This is important as different methods would take different amount of times. Would it not be simpler to just display the amout of people with special requirements, and then the organizer manually email the caterers?"                       | 
| 32     | "I want to see the table layout and seating arrangements, so the tables can be organised"                                                                 |                                                                                                                                                                                                                                                                                       | 
| 33     | I want to be able to see the table plan                                                                                                                   | "Is this the same story as above (number 32)? If not, in which ways are they different?"                                                                                                                                                                                              | 
| 34     | I want to be able to manage table settings                                                                                                                | "Are table settings (such as cuttlery, etc) not provided by catering? If not, what aspect of a table set should be changeable?"                                                                                                                                                       | 
| 36     | "Allocate seats correctly == -people who asked to sit together, VIP tables, High network individuals/spenders near stage"                                 |                                                                                                                                                                                                                                                                                       | 
| 37     | Organize tables where ‰ŰĎbig spenders‰Űť should be seated                                                                                                 | "Is this the same story as above (number 36)? If not, in which ways are they different?"                                                                                                                                                                                              | 
| 40     | I want to organise sub events                                                                                                                             | "What is a ""sub event""? With no detail, it is impossible to accurately estimate the time needed to produce this feature."                                                                                                                                                           | 
| 41     | Create game                                                                                                                                               | This is too vague to accurately estimate a time for.                                                                                                                                                                                                                                  | 
| 51     | I want to see how much is raised so far                                                                                                                   |                                                                                                                                                                                                                                                                                       | 
| 52     | Have a tracker/ live count that updates with donations throughout the event                                                                               | "Is this the same story as above (number 51)? If not, in which ways are they different?"                                                                                                                                                                                              | 
| 53     | I want to track current amount raised                                                                                                                     | "Is this the same story as above (number 52)? If not, in which ways are they different?"                                                                                                                                                                                              | 
| 54     | I want to view current amount raised                                                                                                                      | "Is this the same story as above (number 53)? If not, in which ways are they different?"                                                                                                                                                                                              | 
| 57     | I want to manage staff for the night                                                                                                                      | "What does this involve? Simply viewing locations of staff members, or assigning them to new tasks? This does not seem useful in this application."                                                                                                                                   | 
| 58     | "As a staff member on the night, I want to be able to communication with other staff members, so we can deal with issues that might arise."               | "Is this a form of text or voice chat between staff? There are many messaging apps that can be used instead, as developing this system would take weeks, and achieve only achieve something fractionally as good."                                                                    | 
| 60     | Follow up with attendees (queries)                                                                                                                        | "Is this the same story as number 21? if not, how are they distinct?"                                                                                                                                                                                                                 | 
| 62     | Create an account/register                                                                                                                                | "Why would an attendee ever want to fill out a registration form? You could instead a confirmation number on each ticket, that allows access to a profile, where one could bid, input diatary requirements, etc."                                                                     | 
| 63     | Login to account                                                                                                                                          | "Why have a login at all for attendees, surely that happens when you arrive to an event."                                                                                                                                                                                             | 
| 64     | I want to be able to add my details to the site                                                                                                           | "Is this a part of registration, if a attendee is required to register?"                                                                                                                                                                                                              | 
| 66     | I want to view event information so I can decide if I want to attend                                                                                      |                                                                                                                                                                                                                                                                                       | 
| 67     | "I want to be able to easily see details fo the event ‰ŰŇ time, venue, etc"                                                                               | "Is this the same as above (number 66)? If not, how are they distinct?"                                                                                                                                                                                                               | 
| 68     | Look at dinner menu                                                                                                                                       | "Is this the same as above (number 67)? If not, how are they distinct?"                                                                                                                                                                                                               | 
| 69     | I would like to buy a ticket to attend the event                                                                                                          | "Does this include payment, or is payment done at the end, eg. an invoice is sent to the attendee."                                                                                                                                                                                   | 
| 70     | "I want to be able to purchase tickets througha simple online site, so I can attend the event"                                                            | "Is this the same as above, except with payment?"                                                                                                                                                                                                                                     | 
| 72     | "Check date, time, seat at event, so I can arrive on time and find my seat easily"                                                                        | Is finding their seat within the scope of this application. Does this mean a map and GPS coordinates for the venue must also be supplied?                                                                                                                                             | 
| 73     | Accept invitation                                                                                                                                         | "Is this if attendees are given tickets, or do they have to pay now, or have they already paid?"                                                                                                                                                                                      | 
| 74     | I want to be able to RSVP so my intention to attend is noted                                                                                              | "Have they already paid? Or, has a ticket been gifted to them? Or are they for free tickets?"                                                                                                                                                                                         | 
| 75     | I want to be able to register for an event                                                                                                                | Is this the same are RSVP or is this the same story as number 62                                                                                                                                                                                                                      | 
| 76     | "I would like to be able to cancel my reservation, if I can‰ŰŞt attend"                                                                                   | "Does this require issueing a refund, or notifying the organisers?"                                                                                                                                                                                                                   | 
| 80     | I want to be able to receive notifications                                                                                                                | "What form of notifications? Since this is a web app and not a native mobile app, only email notifications are possible."                                                                                                                                                             | 
| 81     | I would like to choose my seating if I‰ŰŞm going with friends                                                                                             |                                                                                                                                                                                                                                                                                       | 
| 82     | "I would like to pick seats, in order to sit beside friends"                                                                                              | Is this the same as above (number 81)?                                                                                                                                                                                                                                                | 
| 88     | I want to be notified when new events are released                                                                                                        | By email?                                                                                                                                                                                                                                                                             | 
| 92     | I want to be able to make payments                                                                                                                        | "Are we supposed to do any of the above (eg ticket selling, e-ticketing) without payments if this story is not taken?"                                                                                                                                                                | 
| 95     | Be able to pledge through a webapp                                                                                                                        | "Is this the same story as number 90, as this is going to be a web app anyway?"                                                                                                                                                                                                       | 
| 97     | I want to see how much has been raised so far.                                                                                                            | Is this the same as number 52?                                                                                                                                                                                                                                                        | 
| 98     | I want to be able to see a live count of the funds raised so I can see how close the event is to raising money for specific needs of the charity          | Is this the same as above (number 97)?                                                                                                                                                                                                                                                | 
| 99     | I would like to be able to see the running total for the current event                                                                                    | Is this the same as above (number 98)?                                                                                                                                                                                                                                                | 
| 100    | I want to know who much was raised at an event that I attended                                                                                            | Is this the same as above (number 98)?                                                                                                                                                                                                                                                | 
| 101    | I want to be able to see how my donation has helped the charity                                                                                           | Is this to do with where the money goes once donated? Is this the same for every donor?                                                                                                                                                                                               | 
| 104    | "I want to be able to bid in the auction, because I want the item"                                                                                        |                                                                                                                                                                                                                                                                                       | 
| 106    | Be able to bid through a webapp                                                                                                                           | Is this the same as 95 or 104?                                                                                                                                                                                                                                                        | 
| 108    | "As a guest, I have coat (or other items) that I need to check in and I need an identification code to get it back"                                       | "Maybe just use a ticket confirmation code, that is printed on their ticket? Or their name. For a classy event, you hardly want people taking out their phone, visiting a website, logging in and then showing a confirmation number to on their screen just to get their coat back." | 
| 109    | I want to be able to get the attention of the staff waiting on tables if there are any issues that need to be dealt with                                  | On your phone?                                                                                                                                                                                                                                                                        | 
| 110    | I would like to be able to call table service                                                                                                             | On your phone?                                                                                                                                                                                                                                                                        |

**Comment by andrewbutterfield** [comment]
Response part 1

I think I can be added as a collaborator, and then I can unwatch your repo so I only see issues were I am explicitly @mentioned.

**Comment by andrewbutterfield** [comment]
Response part 2 - the backlog will match the available time. Note that while about 25% of the elapsed (wall clock) time  for Release one has gone, you should only be working approx 1/3rd of each week, so you can concentrate on your FYPs now while waiting for things to settle here....

**Comment by andrewbutterfield** [comment]
Response part 3 - deferred

We'll get back to you on the individual queries once we have some prioritisation done. Note that the attendee features are the most vague. 

Some apparently different features are very similar - we'll probably just take one of them.  Just give any you think are the same the same estimate.
